packages/client/App.tsx

```diff
@@ -35,6 +37,16 @@ const GET_COORDINATES = gql `
         profileText
         profilePicture
         images
+        humanRightsScore
+        carbonScore
+        certificateScore
+        envProtectionScore
+        diversityScore
+        productSafetyScore
+        equalPayScore
+        taxScore
+        dataPrivacyScore
+        charitableScore
```

Comment:
Do we need all this data this early on?

Reply:
I figured it was easier to query everything we need in one query rather than query the database an excessive number of times.